README.md

```diff
@@ -275,7 +270,9 @@ func main() {
   discoveryService, discoveryServiceErr := discovery.NewDiscoveryV1(&discovery.DiscoveryV1Options{
     URL:       "YOUR SERVICE URL",
     Version:   "2018-03-05",
-    IAMApiKey: "YOUR IAM API KEY",
+	Authenticator: &core.IamAuthenticator{
+		Apikey: "YOUR APIKEY",
+	},
```

Comment:
is indentation off here? i see it in a few examples

Reply:
in Go, each indent level is supposed to be a tab character